include/z64.h

```diff
@@ -1805,13 +1815,12 @@ typedef struct {
     /* 0x14 */ Gfx* dList;
 } VisMono; // size = 0x18
 
-// Vis...
 typedef struct {
     /* 0x00 */ u32 useRgba;
     /* 0x04 */ u32 setScissor;
     /* 0x08 */ Color_RGBA8_u32 primColor;
-    /* 0x08 */ Color_RGBA8_u32 envColor;
-} struct_80166500; // size = 0x10
+    /* 0x0C */ Color_RGBA8_u32 envColor;
+} VisZbuf; // size = 0x10
```

Comment:
Should this be `VisZBuf` since we use "z-buffer" and not the single word variant? (eg. we have `gZBuffer` instead of `gZbuffer`)

Reply:
Ah, I think I was using `prerender` as reference which consistently uses `zbuf`, I can change if capital b is preferred.

Reply:
Afaik we agreed to use z-buffer and thus capital B everywhere, tho it's not very consistent yet (similar situation with frame buffer which is still treated as a single word in a few places atm)